microsoft-edge/devtools-guide-chromium/sources/index.md

```diff
@@ -22,62 +22,231 @@ keywords: microsoft edge, web development, f12 tools, devtools
 :::image-end:::  
 
-## Edit CSS and JavaScript  
+A file that's in a Workspace is indicated by a green dot on the file in the **Filesystem** tab.  A green dot is also added on the corresponding file that is returned by the server, in the **Page** tab:
 
-Use the **Editor** pane to edit CSS and JavaScript.  DevTools updates the page to run your new code.  For example, if you edit a CSS file by adding the style rule below:
+:::image type="complex" source="../media/sources-filesystem-tab-page-tab-dots.msft.png" alt-text="The Page tab, with green dots indicating corresponding files are in a local Workspace" lightbox="../media/sources-filesystem-tab-page-tab-dots.msft.png":::
+   The Page tab, with green dots indicating corresponding files are in a local Workspace
+:::image-end:::  
 
-```css
-.metadata.page-metadata {
-    color: red;
-}
-```
+Using a Workspace saves the changes that you make in DevTools to your file system.
+
+By using a Workspace, DevTools is able to be used as your code editor and Integrated Development Environment (IDE) for your source files.  When you edit the copy of the front-end code that's returned by the server, the edits are also applied to your local source code.  For other users to see your changes, you redeploy your changed source files (front-end code) to the server.
+
+A Workspace works well when the JavaScript code that's returned by the server is the same as your local JavaScript source code that you maintain, rather than performing minification, transformation, or other compiling of JavaScript code.
+
+When using a Workspace, changes to your front-end code are preserved across page refreshes.
+
+For more information, navigate to the tutorial [Edit files with Workspaces][DevtoolsGuideChromiumWorkspacesIndex].
+
+#### Sources tool without a Workspace
+
+In contrast to using a Workspace, when you use the Sources tool in the default way, you edit a transient copy of the front-end code that's returned by the web server.  This non-Workspace approach supports fast iterations when you are prototyping JavaScript code and don't want to go through the build process every time you want to view the rendered result of a code edit.
+
+By default, when you edit a file in the **Sources** tool, those changes are lost when you refresh the page.  In the default approach, the **Sources** tool works with a copy of the client-side resources that are returned by the web server.  When you modify these client-side files that are returned by the server, the changes don't persist, because you didn't change the source files.
+
+When you find a fix that works for the client-side JavaScript that is returned by the server, you then need to apply that fix, or an equivalent of that fix, in your actual source code and then re-deploy to the server.
+
+#### Mapping minified code to your source code, to work with readable source code
+
+<!-- From a user point of view, is Workspace related to mapping minified code to source code? -->
```

Comment:
@zoherghadyali    From a user point of view, is Workspace related to mapping minified code to source code?